### DISCUSSION TITLE
Review System to Rate Event Organizers

### DISCUSSION THREAD

**Discussion by GabyMarr** [uprm-inso4101-2024-2025-s1/semester-project-campus-resource-and-event-management#3] [Ideas]
Idea: Creating a Review System to allow users to rate events that are displayed. The ways I think we could implement this is would have two separate ways to rate/review. 

1. Rating System for events
2. Rating System for event organizers

Purpose: Allows great event organizers to gain appreciation and notority for their efforts in the industry.

**Comment by JeanSanchezFelix** [comment]
Please reformat your discussion post. I don't see a clear problem statement or proposal. Please remember this is a professional environment and informal language is unhelpful when it comes to setting goal and task.
